tests/chainer_tests/test_variable.py

```diff
@@ -1323,22 +1323,7 @@ def test_backward_gpu(self):
         self.check_backward(cuda.to_gpu(self.x))
 
 
-@testing.parameterize(
-    {'x_shape': None, 'dtype': None, 'repr': 'variable(None)',
-     'str': 'variable(None)'},
-    {'x_shape': (2, 2,), 'dtype': np.float16,
-     'repr': 'variable([[ 0.,  1.],\n          [ 2.,  3.]])',
-     'str': 'variable([[ 0.  1.]\n          [ 2.  3.]])'},
-    {'x_shape': (2, 2,), 'dtype': np.float32,
-     'repr': 'variable([[ 0.,  1.],\n          [ 2.,  3.]])',
-     'str': 'variable([[ 0.  1.]\n          [ 2.  3.]])'},
-    {'x_shape': (2, 2,), 'dtype': np.float64,
-     'repr': 'variable([[ 0.,  1.],\n          [ 2.,  3.]])',
-     'str': 'variable([[ 0.  1.]\n          [ 2.  3.]])'},
-    {'x_shape': (3,),  'dtype': np.float32,
-     'repr': 'variable([ 0.,  1.,  2.])', 'str': 'variable([ 0.  1.  2.])'},
-)
-class TestUnnamedVariableToString(unittest.TestCase):
+class _TestUnnamedVariableToStringBase(object):
```

Comment:
I think the prefix `_` is not necessary because this is a test code.

Reply:
Actually I first did that, but pytest discovers classes starting with `Test` even if it does not inherit `unittest.TestCase`.
https://pytest.readthedocs.io/en/reorganize-docs/new-docs/user/naming_conventions.html

Reply:
Oh, then, how about not using `Test` as prefix?

Reply:
OK, I fixed to `UnnamedVariableToStringTestBase`.
I found some tests using `*TestBase` class name.
https://github.com/chainer/chainer/blob/master/tests/chainer_tests/test_function_node.py#L502
https://github.com/chainer/chainer/blob/master/tests/chainer_tests/optimizers_tests/test_optimizers_by_linear_model.py#L82